bokeh/embed.py

```diff
@@ -33,7 +33,7 @@
 def _wrap_in_function(code):
     # indent and wrap Bokeh function def around
     code = "\n".join(["    " + line for line in code.split("\n")])
-    return 'Bokeh.$(function() {\n%s\n});' % code
+    return code #'Bokeh.$(function() {\n%s\n});' % code
```

Comment:
Given how we generate HTML, this wrapper is unnecessary. I actually wanted to remove it recently. However, it may be useful if user wants to write his own HTML.